makefile

```diff
@@ -13,10 +13,30 @@ omim.sssom.tsv: omim.json
 	sssom parse omim.json -I obographs-json -m data/metadata.sssom.yml -o omim.sssom.tsv
 	make cleanup
 
+mondo_exactmatch_omim.sssom.tsv:
+	wget "http://purl.obolibrary.org/obo/mondo/mappings/mondo_exactmatch_omim.sssom.tsv" -O $@
+
+mondo_exactmatch_omimps.sssom.tsv:
+	wget "http://purl.obolibrary.org/obo/mondo/mappings/mondo_exactmatch_omimps.sssom.tsv" -O $@
+
+%.sssom.owl: %.sssom.tsv
+	sssom convert $< -O owl -o $@
+
 # More commands / goals  -----------------------------------------------------------------------------------------------
 # Create mapping artefact(s)
-omim.json:
-	robot convert -i omim.ttl -o omim.json
+omim.json: omim.owl
+	robot convert -i $< -o omim.json
+
+# Create OWL artefact, but adding HGNC links alongside the OMIM genes and
+# Mondo mappigs alongside the OMIM diseases
```

Comment:
Typo: `mappigs`

Reply:
Use suggestion mode for these!

<img width="471" alt="image" src="https://github.com/monarch-initiative/omim/assets/7070631/dcac9648-97d7-4d13-ad9e-94715833ab57">

Reply:
ah yes